examples/WORKSPACE

```diff
@@ -97,6 +106,7 @@ stack_snapshot(
     snapshot = "lts-14.0",
     vendored_packages = {"split": "@split//:split"},
     deps = ["@zlib.dev//:zlib"],
+    tools = [ "@stdenv//:bin/{}".format(tool) for tool in ["sh", "grep", "sed", "awk"]],
```

Comment:
It's going to be quite a cost for all Nixpkgs users if they have to specify this any time they use `stack_snapshot`. What's the reason for this and are there no alternatives?

Reply:
Yes I agree, I've included this out of despair, but I'd really like another solution. The issue is that some cabal packages need these tools at build time (the ones with a configure script), but afaik there's no clean way of getting them in bazel (It really sucks that there's no "bash" toolchain :confused: ).

Outside of NixOS, these tools are incidentally put into the PATH by https://github.com/tweag/rules_haskell/blob/2c4dc52a4e60683a0d0ff863ed6b19b48e0bc734/haskell/private/cabal_wrapper.py.tpl#L57 , but that's an awful hack that doesn't work on NixOS.

If you know any way of giving the rule access to a standard bash environment, I'm all ears